cli/src/template/android/app/src/main/java/com/example/moproapp/MainActivity.kt

```diff
@@ -14,6 +14,7 @@ import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.unit.dp
+import com.mopro.mopro_app.ui.theme.MoproappTheme
```

Comment:
try to avoid this

Reply:
Could you eleborate more?

Reply:
The `MoproappTheme` is a customized `{NAME}Theme`
If we changed the project, the Theme name and the path would be different

We don't want to maintain such a customized thing
https://zkmopro.org/docs/setup/android-setup

Reply:
fixed in [fc8bd82](https://github.com/zkmopro/mopro/pull/399/commits/fc8bd822cdf3b5c8cc5be9abac87752e89e6e4f5)